File: id-card-toolkit-ios-sdk-v3.1.6/id-card-toolkit-ios-sdk-v3.1.6/samples/IDCardToolkitObjCApp/IDCardToolkitObjCApp/Other files/AlertView.m
//
//  AlertView.m
//
//
//  Created by Federal Authority For Identity and Citizenship  on 1/19/17.
//  Copyright © 2017 Federal Authority For Identity and Citizenship. All rights reserved.
//

#import "AlertView.h"

@implementation AlertView

+ (void)showAlertTitle:(NSString *)title withMessage:(NSString *)message onView:(UIViewController *)viewController
{
    dispatch_async(dispatch_get_main_queue(), ^{
   
    UIAlertController * alert=   [UIAlertController
                                  alertControllerWithTitle:title
                                  message:message
                                  preferredStyle:UIAlertControllerStyleAlert];
    
    UIAlertAction* okButton = [UIAlertAction
                               actionWithTitle:@"OK"
                               style:UIAlertActionStyleDefault
                               handler:^(UIAlertAction * action)
                               {
                                   [alert dismissViewControllerAnimated:YES completion:nil];
                                   
                               }];
    
    [alert addAction:okButton];
    
    [viewController presentViewController:alert animated:YES completion:nil];
    
   });// update UI main queue
}
@end

File: id-card-toolkit-ios-sdk-v3.1.6/id-card-toolkit-ios-sdk-v3.1.6/samples/ICASwiftApp/ICASwiftApp/ConversionClass/AlertView.swift
//
//  AlertView.swift
//  EIDAToolkitSwiftTest
//
//  Created by  on 1/19/17.
//  Copyright © 2017 Emirates Identity Authority. All rights reserved.
//

import UIKit

class AlertView: NSObject {
    
  class func showAlertTitle(_ title :String,withMessage message:String,onView viewController:UIViewController)  {
    
        DispatchQueue.main.async(execute: {() -> Void in
        let  alert = UIAlertController(title: title, message: message, preferredStyle: .alert)
        let okButton = UIAlertAction(title: "OK", style: .default, handler: {(_ action: UIAlertAction) -> Void in
            alert.dismiss(animated: true, completion:nil)
        })
        alert.addAction(okButton)
        viewController.present(alert, animated: true, completion: nil)
        })// update UI main queue
    }// showAlertTitle()

}
